static/app/views/eventsV2/results.tsx

```diff
@@ -121,6 +122,13 @@ class Results extends Component<Props, State> {
 
   componentDidMount() {
     const {organization, selection, location} = this.props;
+    if (location.query.fromMetric) {
+      this.setState({showMetricsAlert: true});
```

Comment:
should we clear the alert when the user modifies the query?

Reply:
Good question, I'll look into it and we can add an another pr!